assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -71,19 +72,20 @@ export default function PropertySelect() {
 	);
 
 	const { selectProperty } = useDispatch( MODULES_ANALYTICS );
+	const viewContext = useContext( ViewContextContext );
 	const onChange = useCallback(
 		( index, item ) => {
 			const newPropertyID = item.dataset.value;
 			if ( propertyID !== newPropertyID ) {
 				selectProperty( newPropertyID, item.dataset.internalId ); // eslint-disable-line sitekit/acronym-case
-				trackEvent(
-					'analytics_setup',
-					'property_change',
-					newPropertyID
-				);
+				const action =
+					newPropertyID === PROPERTY_CREATE
+						? 'change_property_new'
+						: 'change_property';
+				trackEvent( `${ viewContext }_analytics`, action, 'ga4' );
```

Comment:
It should use the `ua` event label here.

Reply:
https://github.com/google/site-kit-wp/pull/4321/commits/45bc9df81383e01da43cd8f131d007066881d922